kano_settings/system/advanced.py

```diff
@@ -112,16 +112,18 @@ def create_empty_hosts():
 
 def add_safesearch_blacklist(hosts):
     '''
-    Prevents google surfing by adding the worldwide sites to the blacklist
+    Prevents surfing to generic search engine sites by adding them to the blacklist
     '''
-    url_pattern='127.0.0.1\twww.google.{}\n'
-    www_pattern='127.0.0.1\tgoogle.{}\n'
 
+    url_pattern='127.0.0.1\t{}\n'
+    www_pattern='127.0.0.1\twww.{}\n'
+
+    # Block Google sites associated to each country (ISO 3166-1 alpha-2)
     for country in pycountry.countries:
-        hosts.append(url_pattern.format(country.alpha2.lower()))
-        hosts.append(www_pattern.format(country.alpha2.lower()))
+        hosts.append(url_pattern.format('google.{}'.format(country.alpha2.lower())))
```

Comment:
this can be added to the other `for countr` loop, so `google` is treated as the other search engines rather than as an exception